include.js

```diff
@@ -1,4 +1,11 @@
 let cards = [
+  {
+    artName: 'Rotating Cube',
+    pageLink: './Art/Rishi/cube.html.html',
```

Comment:
This needs to be `'./Art/Rishi/cube.html'` to match the filename in your folder.